integration_tests_v2/tests/test_volume_anomalies.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime, timedelta
+
+from data_generator import DATE_FORMAT, generate_dates
+from dbt_project import DbtProject
+
+TIMESTAMP_COLUMN = "updated_at"
+DBT_TEST_NAME = "elementary.volume_anomalies"
+DBT_TEST_ARGS = {"timestamp_column": TIMESTAMP_COLUMN}
+
+
+def read_status(dbt_project: DbtProject, test_id: str):
+    results = dbt_project.read_table(
+        "elementary_test_results",
+        where=f"table_name = '{test_id}'",
+        column_names=["status"],
+    )
+    return results
+
+
+def test_anomalyless_table_volume_anomalies(request, dbt_project: DbtProject):
+    test_id = request.node.name
```

Comment:
Consider making a fixture for the `test_id` instead of every test receiving a request object

Reply:
That's just so much better 🤦🏽